Taskfile.yml

```diff
@@ -33,7 +33,7 @@ tasks:
   postpull:
     cmds:
       - task: 'workspaces:clean'
-      - FLAGS_FILE=.env.local task workspaces:setenv
+      - FLAGS_FILE=.env task workspaces:setenv
       - task: 'workspaces:build'
```

Comment:
@niclim I'm sneaking this in, as this little tweak just cost me a whole bunch of head scratching, not understanding why Optic wouldn't compile with the removal buttons showing up. I enabled the feature flag in my local env, yet, wtf? Only to realise that my local `.env` is being overwritten!

A developer's local `.env` should never be overwritten, it's how one developer can enable some features (possibly completely local ones!) or adapt building Optic to their local machine (like an ARM Mac!), while not bothering others with that. If anyone want to mimic a certain environment, they have to opt-in.

Reply:
Ah yeah that makes sense - I definitely overlooked this when I implemented this (i.e. FFs, and custom overrides). I still think we might want to have a "default" if `.env` isn't specified, given our env variables do drive some functionality (share). I think there's better env variable strategies that we could spend some time on.

Reply:
I guess a default config could be copied in place if no local config exists, making that a bit smoother for first time setup / fresh clones. Even better would be a _merge_ instead of a replace, adding in any new settings that aren't set locally.

Reply:
Ah yeah, that was the original intent, just implemented poorly - i'll take a stab at this sometime in the future (will matter when we have new team members / open source contributors)